react-router/lib/createMemoryHistory.d.ts

```diff
@@ -1,3 +1,8 @@
+// Type definitions for react-router v2.0.0
```

Comment:
Yup sounds good. I will update the definition-tester and then update the PR with the remaining errors.